README.md

```diff
@@ -1,31 +1,148 @@
+- [uxit](apps/uxit): The public dashboard of the The Filecoin Foundation User Experience Improvement Team, live at [uxit.fil.org](https://uxit.fil.org)
+
+### Packages
+
+You can find the packages in the `packages` directory. These packages include components, hooks, types, utility functions and configuration shared accross the applications.
+
+> [!NOTE]
+> They are are [just-in-time](https://turborepo.com/docs/core-concepts/internal-packages#just-in-time-packages) packages, meaning they are built by the application that uses them. They do not produce a compiled JavaScript output and are not published anywhere.
 
 ## Getting Started
 
-To get started, clone the repository and install the dependencies:
+### Installation
+
+To get started, clone the repository:
 
 ```bash
-git clone https://github.com/FilecoinFoundationWeb/filecoin-foundation
+git clone https://github.com/FilecoinFoundationWeb/filecoin-foundation.git
 cd filecoin-foundation
+```
+
+Then, install the dependencies:
+
+```bash
 npm install
 ```
 
-## Development
+At this point you should be able to run the development server for any of the applications.
+
+### Development
 
-To start the development server, run `npm run dev`, which is an alias for `turbo dev`. This command will start the development server for all applications and packages in the monorepo.
+Running `npx turbo dev` from the root of the monorepo will call the `dev` script for all applications and packages in the monorepo, as defined in each `package.json`. In our case, this will start the development server for all applications in our monorepo, packages do not have a `dev` script.
+
+In 99% of cases, you only want to start the development server for one application at a time. To so start do so, you should run `turbo dev` with a filter flag.
+
+For instance, to start the development server of the `ffdweb-site` application, run:
+
+```bash
+npx turbo dev --filter=ffdweb-site
+```
+
+You can also use the shorthand syntax:
+
+```bash
+npx turbo ffdweb-site#dev
+```
+
+> [!NOTE]
+> When using npm commands, such as `npm install`, the `--workspace` flag refers to the path of the application or package, relative to the root of the monorepo. When using turbo commands, such as `turbo dev`, the `--filter` flag refers to the name of the application or package, as defined in each `package.json` file.
+
+### Adding npm dependencies
+
+The `npm install <dependency>` command will install dependencies for all applications and packages in the monorepo. Generally, you want to add a dependency inside a specific application or package. To do so, run:
+
+```bash
+npm install <dependency> --workspace=<application/package>
+```
+
+> [!NOTE]
+> The install commands should be run from the root of the monorepo. Do not install dependencies inside the application or package directory, otherwise Turborepo will not be not aware of it and will not resolve the dependencies properly.
```

Comment:
```suggestion
> The install commands should be run from the root of the monorepo. Do not install dependencies inside the application or package directory, otherwise Turborepo will not be aware of it and will not resolve the dependencies properly.
```